.github/workflows/cicd-main.yml

```diff
@@ -156,12 +156,14 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - name: Run tests ${{ matrix.folder }} (CPU)
-        timeout-minutes: 20
+        timeout-minutes: 30
         run: |
           cd InternVideo && patch -p1 < ../external/intern_video2_multimodal.patch && cd .. && sed -i "/InternVideo/d" .gitignore
           uv sync --link-mode copy --locked --extra audio_cpu --extra text_cpu --extra video_cpu --group test
           uv add InternVideo/InternVideo2/multi_modality
-          uv run coverage run --branch --source=nemo_curator -m pytest -v tests/${{ matrix.folder }} -m "not gpu"
+          FOLDER="${{ matrix.folder }}"
+          FOLDER="${FOLDER/stages-/stages/}"
+          uv run coverage run --branch --source=nemo_curator -m pytest -v "tests/$FOLDER" -m "not gpu"
```

Comment:
"Upload artifacts" step complains if there is a slash in the folder name and experiencing weirdness when I tried to add a `replace` function. Editing it this way seems to work fine though.